ciao-launcher/tests/ciao-launcher-server/server.go

```diff
@@ -491,6 +495,8 @@ func createConfigFile(confPath string) error {
 	conf.Configure.Launcher.MemoryLimit = memLimit
 	conf.Configure.Launcher.ComputeNetwork = []string{computeNet}
 	conf.Configure.Launcher.ManagementNetwork = []string{mgmtNet}
+	conf.Configure.Storage.SecretPath = secretPath
```

Comment:
And check to see if the values received from the command line are "" or not only performing the assignment if they are !="".